public/manifest.json

```diff
@@ -0,0 +1,36 @@
+{
+  "manifest_version": 3,
+  "name": "Badge Printer",
+  "version": "1.0.0",
+  "action": {
+    "default_title": "Print badge",
+    "default_popup": "popup.html"
+  },
+  "permissions": [
+    "declarativeContent",
+    "scripting",
+    "storage"
+  ],
+  "host_permissions": [
+    "https://manage.webconnex.com/"
+  ],
```

Comment:
Do we want to add the regfox URLs in here as well just to future proof it? Manifest changes on things like that tend to invoke re-review when submitting extensions to the extension store.